Arguments: 
  /usr/bin/node /usr/bin/yarn publish

PATH: 
  /usr/local/sbin:/usr/local/bin:/usr/bin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl

Yarn version: 
  1.3.2

Node version: 
  9.3.0

Platform: 
  linux x64

npm manifest: 
  {
    "name": "sphynx",
    "version": "2.0.1",
    "description": "Middleware for authenticating microservices",
    "main": "app.js",
    "keywords": [
      "microservices",
      "microservice",
      "docker",
      "authentication",
      "security",
      "jwt",
      "jsonwebtokens"
    ],
    "author": "Ulisses Caon",
    "license": "MIT",
    "dependencies": {
      "axios": "^0.17.1"
    }
  }

yarn manifest: 
  No manifest

Lockfile: 
  No lockfile

Trace: 
  Error: https://registry.yarnpkg.com/-/user/org.couchdb.user:caonUlisses: Unknown error while authenticating
      at Request.params.callback [as _callback] (/usr/lib/node_modules/yarn/lib/cli.js:62098:18)
      at Request.self.callback (/usr/lib/node_modules/yarn/lib/cli.js:123085:22)
      at Request.emit (events.js:159:13)
      at Request.<anonymous> (/usr/lib/node_modules/yarn/lib/cli.js:124068:10)
      at Request.emit (events.js:159:13)
      at IncomingMessage.<anonymous> (/usr/lib/node_modules/yarn/lib/cli.js:123988:12)
      at Object.onceWrapper (events.js:254:19)
      at IncomingMessage.emit (events.js:164:20)
      at endReadableNT (_stream_readable.js:1062:12)
      at process._tickCallback (internal/process/next_tick.js:152:19)
